int32 power_value
---
int32 power_percent_new
